recipes/maddy/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class MaddyConan(ConanFile):
+    name = "maddy"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/progsource/maddy"
+    description = (
+        "open-source, maddy is a C++ Markdown to HTML header-only parser library."
+    )
+    topics = ("maddy", "markdown", "header-only")
+    license = "MIT"
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
```

Comment:
```suggestion

    def validate(self):
        if self.settings.compiler.get_safe("cppstd"):
            tools.check_min_cppstd(self, 14)

```

It's a C++14 library.